### DISCUSSION TITLE
PsPAS Connection error

### DISCUSSION THREAD

**Discussion by winbro** [pspete/psPAS#411] [General]
Help appreciated:

code: Trying to connect PVWA,

            #Connect to PVWA
            $cred = Get-Credential 
            New-PASSession -Credential $cred -BaseURI $PVWA -type CyberArk  

 got following error:

WARNING: 2022/08/16 13:49:26 warn :   error :  Runtime Error body font family Verdana font weight normal font size 7em color black p font family Verdana font weight normal color black margin top 5px b font family Verdana font weight bold color bl
ack margin top 5px H1 font family Verdana font weight normal font size 18pt color red H2 font family Verdana font weight normal font size 14pt color maroon pre font family Consolas Lucida Console Monospace font size 11pt margin 0 padding 0 5em li
ne height 14pt marker font weight bold color black text decoration none version color gray error margin bottom 10px expandable text decoration underline font weight bold color navy cursor pointer media screen and max width 639px pre width 440px o
verflow auto white space pre wrap word wrap break word media screen and max width 479px pre width 280px Server Error in PasswordVault Application Runtime Error Description An exception occurred while processing your request Additionally another e
xception occurred while executing the custom error page for the first exception The request has been terminated

**Comment by winbro** [comment]
PS C:\>Get-Module -ListAvailable psPAS

ModuleType Version    Name                                ExportedCommands                                                                                                                                                                           
---------- -------    ----                                ----------------                                                                                                                                                                           
Script     5.3.69     psPAS                               {New-PASSession, Close-PASSession, Add-PASPublicSSHKey, Get-PASPublicSSHKey...}

**Comment by pspete** [comment]
Believe that is what is observed when the "/PasswordVault/" suffix is included in the BaseUri value being used.

https://pspas.pspete.dev/commands/New-PASSession#-baseuri

**Reply by winbro** [reply]
Thanks you so much Pspete.  it worked without the suffix
